output/schema/validation-errors.json

```diff
@@ -482,6 +482,12 @@
         "response definition cluster.put_component_template:Response / body / instance_of - Non-leaf type cannot be used here: '_types:AcknowledgedResponseBase'"
       ]
     },
+    "connector.delete": {
+      "request": [],
+      "response": [
+        "response definition connector.delete:Response / body / instance_of - Non-leaf type cannot be used here: '_types:AcknowledgedResponseBase'"
```

Comment:
This issue is common for other endpoints that deal with `delete` action - should I change something here?

Reply:
No, you can leave it as is.